### DISCUSSION TITLE
ASCII polar notation

### DISCUSSION THREAD

**Discussion by mofosyne** [josdejong/mathjs#2698] [Ideas]
So how would one do phasor/polar notations in text? I tend to write like below (depending on context) 

```
polar = 13 /_ 22.6 ^deg
```

Also could want to display in polar notation rather than Cartesian. e.g.

```
polarNotation(polar)
      := 13 /_ 22.6 ^deg
```

Or maybe we should just display both as standard? Or as a display switch? Or as a function like verbose() ?

```
polar
     := 12.002 + 4.996i = 13 /_ 22.6 ^deg
```

All calculations would still be in Cartesian.

---

Hmmmm... Maybe when declaring/defining a variable, it could be more flexible to parse notations like above?

**Comment by mofosyne** [comment]
Oh, btw just noted that `/_` is supported by asciimath. Maybe they got a few other things we can lift, in terms of syntactic sugar variable declaration (like polar notations).

**Comment by josdejong** [comment]
Thanks for your suggestion.

We can indeed implement an operator `/_`, this will not give conflicts. Notation would then be `13 /_ 22.5 deg` (without the `^` before deg). 

I'm not sure how widely used this is, but it's interesting that this same notation is used in asciimath.

**Comment by mofosyne** [comment]
Mostly used in power engineering, physics and anything that refers to vectors in a polar notation. 

E.g. cannon ball firing.
